tollway/vehicle.py

```diff
@@ -52,14 +53,12 @@ def get_tollways(html: Optional[str] = None) -> dict:
 
         tables.append(tolls)
 
-    united_states_tollways = dict(
-        (state, tolls) for state, tolls in zip(states, tables) if tolls
-    )
+    united_states_tollways = dict((state, tolls) for state, tolls in zip(states, tables) if tolls)
```

Comment:
read somewhere {} is a bit faster than dict (not significant though), more importantly I find it slightly more readable to write it like this = more compact:

```
united_states_tollways = {state: tolls for state, tolls in zip(states, tables) if tolls}
```

Reply:
agreed. It is a lot more readable like this.